lib/oli/authoring/publishing.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Oli.Authoring.Publishing do
```

Comment:
This module name does not make sense given the contents within it.  The functions within it are Delivery functions (e.g. "find instructors enrolled in a section") but the Module name here involves Authoring and Publishing. 

I would eliminate this module and move these functions to a pure Delivery module.